manifests/init.pp

```diff
@@ -36,6 +36,7 @@
   # interval in hours, purging enabled when >= 1
   $purge_interval   = 0,
   # log4j properties
+  $log4j_file   = undef,
```

Comment:
Also please note it is up to the user to get this file deployed and that is outside the scope of this module